CTFd/__init__.py

```diff
@@ -114,10 +117,18 @@ def get_source(self, environment, template):
             template = "/".join(["admin", "templates", template])
             return super(ThemeLoader, self).get_source(environment, template)
 
-        # Load regular theme data
-        theme = str(utils.get_config("ctf_theme"))
-        template = "/".join([theme, "templates", template])
-        return super(ThemeLoader, self).get_source(environment, template)
+        # Load regular theme data with potential fallbacks
+        first_cand, *others = ctf_theme_candidates()
```

Comment:
Is there a reason to need to break this out into two lists? I think `themes = list(ctf_theme_candidates())` would work?

Reply:
Ah I see it's for the explicit exception!